### ISSUE TITLE
Error: `DEFAULT value is not allowed when jsonpath is set` when running Dub.co's pipe

### ISSUE THREAD

**Issue by unrenamed** [dubinc/dub#1623]
### Issue Summary

While following [the guide to run Dub.co’s codebase locally](https://dub.co/docs/local-development#step-2-set-up-tinybird-clickhouse-database), I encountered an error when executing `tb push`. The error message indicates that when you specify a column with a jsonpath attribute, you cannot also provide a DEFAULT value for that column.

The error is caused by 4 files:

1. `dub_lead_events.datasource`
2. `dub_links_metadata.datasource`
3. `dub_sale_events.datasource`
4. `dub_webhook_events.datasource`

### Steps to Reproduce

1. Follow the setup instructions provided in the guide up to [Step 2: Set up the Tinybird Clickhouse database](https://dub.co/docs/local-development#step-2-set-up-tinybird-clickhouse-database).
5. After you run `tb push`, wait for the command to fail and observe the error message like on the screenshot below.

### Expected behavior

The command should execute successfully, creating the essential data sources without errors.

### Screenshots

<img width="708" alt="image" src="https://github.com/user-attachments/assets/3f42f943-9ad3-419f-ab9b-afe165bcc5fd">

### Version info
- Tinybird CLI Version 2.0.0 (rev 40c7bee)
- macOS Version 13.5.2 (22G91)

**Comment by nhannv** [comment]
I got the same issue and resolved it by upgrade tinybird-cli: `pip install tinybird-cli==5.13.0`

**Comment by unrenamed** [comment]
@nhannv good catch! It's resolved on my end too—the old CLI version was way outdated.

Should we update the [Dub Docs](https://dub.co/docs/local-development#step-2-set-up-tinybird-clickhouse-database)? For contributors who haven't run the app recently, `pip install tinybird-cli` won’t upgrade an existing install. The docs could clarify whether to install the latest version, how to upgrade, or the minimum supported version for schemas. Thoughts, @steven-tey?
